src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,45 @@
+import { useEffect, useState } from 'react';
+import { Loader } from '../Loader';
+import { PeopleTable } from '../PeopleTable';
+import { getPeople } from '../../api';
+
+import { Person } from '../../types/Person';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[] | null>(null);
+  const [isLoad, setIsLoad] = useState<boolean>(false);
+  const [error, setError] = useState<string | null>(null);
+
+  useEffect(() => {
+    setIsLoad(true);
+    getPeople()
+      .then((peopleFromServer) => setPeople(peopleFromServer))
+      .catch(() => {
+        setError('Something went wrong');
+      })
+      .finally(() => setIsLoad(false));
+  }, []);
+
+  return (
+    <>
+      <h1 className="title">People Page</h1>
+      <div className="block">
+        <div className="box table-container">
+          {isLoad && <Loader />}
+          {error && (
+            <p
+              data-cy="peopleLoadingError"
+              className="has-text-danger"
+            >
+              {error}
+            </p>
+          )}
+          {people && <PeopleTable people={people} />}
+          {!isLoad && !error && !people && (
+            <p data-cy="noPeopleMessage">no people</p>
```

Comment:
Both `error` and `<p data-cy="noPeopleMessage">no people</p>` are errors messages, why do you handle them differently